src/main/java/com/exadel/discountwebapp/category/mapper/CategoryMapper.java

```diff
@@ -3,28 +3,23 @@
 import com.exadel.discountwebapp.category.entity.Category;
 import com.exadel.discountwebapp.category.vo.CategoryRequestVO;
 import com.exadel.discountwebapp.category.vo.CategoryResponseVO;
+import org.modelmapper.ModelMapper;
 import org.springframework.stereotype.Component;
 
 @Component
 public class CategoryMapper {
 
+    private final ModelMapper modelMapper = new ModelMapper();
```

Comment:
add Strict strategy config to the mapper on the constructor level

Reply:
done